src/pynxtools_raman_multiformat/config_file.json

```diff
@@ -0,0 +1,63 @@
+{
+  "/ENTRY[entry]/INSTRUMENT[instrument]/scattering_configuration": "@eln",

```

Comment:
If you want, you can make this a nested dict like this:
```
"/ENTRY[entry]/INSTRUMENT[instrument]": {
  "scattering_configuration": "@eln",
  "beam_incident/wavelength": "@eln",
  "detector_TYPE[detector_ccd]": {
    "detector_channel_type": "@eln",
  },
  ...
}

Reply:
yea, I'll use this later at some other point. Was easier for me to visually see where I work - more similar to a .yaml file. Want to keep  this for now and will extend/explain it later

Reply:
ok, it's mainly to prevent typing the same keys over and over, but both ways are fine